includes/class-woocommerce-delivery-notes.php

```diff
@@ -609,6 +530,87 @@ public function is_woocommerce_activated() {
 				return false;
 			}
 		}
+		/**
+		 * It will delete the tracking option from the database.
+		 */
+		public function ts_reset_tracking_setting() {
+
+			if ( isset( $_GET ['ts_action'] ) && 'reset_tracking' === $_GET ['ts_action'] ) { //phpcs:ignore
```

Comment:
@nidhitatosaniya Since this action makes some changes to the DB, we should implement nonce here.